test/integration/configuration/gcp_inspec_config.rb

```diff
@@ -41,7 +41,13 @@ module GCPInspecConfig
       :gcp_ext_vm_data_disk_name => "gcp-inspec-generic-ext-linux-vm-data-disk",
       :gcp_ext_vm_data_disk_size => "f1-micro",
       :gcp_ext_vm_data_disk_image => "ubuntu-os-cloud/ubuntu-1604-lts",
-      :gcp_storage_bucket_name => "gcp-inspec-storage-bucket-#{(0...15).map { (65 + rand(26)).chr }.join.downcase}",
+      :gcp_storage_bucket_name => "gcp-inspec-storage-bucket-#{(0...25).map { (65 + rand(26)).chr }.join.downcase}",
```

Comment:
Minor but this could surely be pulled out into an `any_string` function

Reply:
agreed, can revisit next time round